src/common/tracing/src/minitrace.rs

```diff
@@ -76,24 +78,43 @@ pub fn init_logging(name: &str, cfg: &Config) -> Vec<Box<dyn Drop + Send + Sync
     // Initialize tracing reporter
     if cfg.tracing.on {
         let name = name.to_string();
-        let jaeger_endpoint = cfg.tracing.jaeger_endpoint.clone();
-        let otlp_reporter = std::thread::spawn(move || {
-            minitrace_opentelemetry::OpenTelemetryReporter::new(
-                opentelemetry_jaeger::new_collector_pipeline()
-                    .with_endpoint(jaeger_endpoint)
-                    .with_reqwest_blocking()
-                    .with_service_name(&name)
-                    .build_collector_exporter::<opentelemetry::runtime::Tokio>()
-                    .expect("initialize jaeger exporter"),
-                opentelemetry::trace::SpanKind::Server,
-                Cow::Owned(opentelemetry::sdk::Resource::default()),
-                opentelemetry::InstrumentationLibrary::new(name, None, None),
-            )
+        let otlp_endpoint = cfg.tracing.otlp_endpoint.clone();
+
+        let (reporter_rt, otlp_reporter) = std::thread::spawn(|| {
+            let rt = tokio::runtime::Runtime::new().unwrap();
```

Comment:
```suggestion
            let rt = tokio::runtime::Runtime::new().worker_threads(2).unwrap();
```

Reply:
This runtime is only for opentelemetry and it seems only use and block one thread.

Reply:
I will modify in anthor PR later, this settting will make the reporter only can use `2 threads `.

Reply:
Fix in:
https://github.com/datafuselabs/databend/pull/13402/files#diff-0cd0738e41abd997179229a48eaed578ee7af4bc2103df9c3e1c743e76a18a6bR85